docs/sdk/paying-for-lit/payment-delegation-db.md

```diff
@@ -0,0 +1,269 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Payment Delegation Database
+
+:::info
+Currently the Payment Delegation Database is only supported on the `habanero` and `manzano` Lit networks. Payment for usage is **not** currently required on the `cayenne` network.
+:::
+
+The Payment Delegation Database is a [smart contract](https://github.com/LIT-Protocol/lit-assets/blob/develop/blockchain/contracts/contracts/lit-node/PaymentDelegation/PaymentDelegationFacet.sol) deployed on Lit's rollup, [Chronicle](../../network/rollup.mdx). Lit's [Relayer server](https://github.com/LIT-Protocol/relay-server) has been updated to provide two new API routes to interface with the Payment Delegation Database contract:
```

Comment:
This is a private repository right now. So we cannot link to it at this time.

Reply:
the same contracts are available in a public repo here, so you can replace the links with these: https://github.com/LIT-Protocol/LitNodeContracts/blob/main/contracts/lit-node/PaymentDelegation/PaymentDelegationFacet.sol